Atomic/scan.py

```diff
@@ -96,6 +97,9 @@ def set_scanner():
 
         scan_type = self.get_scan_type()
 
+        if self.args.profile_id is not None and scan_type != "configuration_compliance":
```

Comment:
Should this type of intelligence be left to the scanning tool rather then hard coded in this way.  Should we allow different scanners to handle profiles differently?